made-with.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default.html
+title: Made With Servo
+---
+
+<style type="text/css">
+
+.made-with-container {
+  gap: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  margin-bottom: 32px;
+}
+
+.made-with-item {
+  display: block;
+  width: 400px;
+  flex: 0 0 400px;
+  border: 2px solid #333;
+  cursor: pointer;
+  color: inherit;
+  text-decoration: inherit;
+}
+
+.made-with-item:hover {
+  background-color: rgba(27, 131, 129, 0.5);
+  background-color: #EAEAEA;
+}
+
+.made-with-text {
+  padding: 20px;
+}
+
+.made-with-text > p {
+  padding: 0;
+}
+
+.made-with-image {
+  aspect-ratio: 2;
+  object-fit: cover;
+  object-position: top left;
+  background-color: transparent;
+}
+
+</style>
+
+<div class="inner-container">
+
+<h1>{{ title }}</h1>
+<p class="subtitle" style="margin-bottom: 32px">
```

Comment:
Remove inline style. It's not needed once 'subpage-content' is added to the correct div as mentioned previously.